providers/Flipt/composer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "open-feature/flipt-provider",
+    "description": "The flipt.io provider package for open-feature",
+    "keywords": [
+        "flipt",
+        "open-feature",
+        "feature flags"
+    ],
+    "homepage": "https://github.com/legoheld/flipt-provider",
```

Comment:
should this be this repo? or our (Flipt's) website?

Reply:
Remove the homepage as its optional. We do it like the other providers.